packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -69,6 +78,35 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
         return undefined;
     }
 
+    private async getLightdashProjectConfig(): Promise<LightdashProjectConfig> {
+        if (!this.dbtProjectDir) {
+            return {
+                spotlight: DEFAULT_SPOTLIGHT_CONFIG,
+            };
+        }
+
+        const configPath = path.join(
+            this.dbtProjectDir,
+            'lightdash.config.yml',
```

Comment:
oh, I see, so we're adding a new lightdash config file. This seems  a bit overcomplicated. Have you considered using ENV variables instead ?

Reply:
We gathered this info from a proposal sent to like ~6 external customers, and they like the managing being done globally (on a separate yml file) + model + metric overrides. 
We are also considering letting them define directories (on the global yml config file) to configure all models in that said directory as most have lots of models/metrics. 
We considered just doing model / level overrides but people want a better devx. You can see more info on this as well on the links shared on the main issue 💪